src/parser/std/pipeline/string/transform.rs

```diff
@@ -1,6 +1,8 @@
 use std::sync::Arc;
 use crate::core::pipeline::item::Item;
 use crate::core::pipeline::items::string::transform::ellipsis::EllipsisItem;
+use crate::core::pipeline::items::string::transform::uppercase::UpperCaseItem;
```

Comment:
Rename this file to `to_upper_case`, rename this item to `ToUpperCaseItem`

Reply:
Sorry I just saw it now, I've fixed it.